package.json

```diff
@@ -87,18 +87,17 @@
     "@visx/shape": "^3.5.0",
     "@visx/tooltip": "^3.3.0",
     "assimpjs": "^0.0.7",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-lodash": "^3.3.4",
+    "babel-loader": "^9.2.1",
     "bottleneck": "^2.19.5",
     "catalog-converter": "^0.0.9",
     "classnames": "^2.3.1",
     "clipboard": "^2.0.8",
-    "commander": "^11.1.0 ",
+    "commander": "^12.1.0 ",
     "copy-webpack-plugin": "^6.4.0",
     "core-js": "^3.1.4",
     "create-react-class": "^15.7.0",
-    "css-loader": "^2.1.0",
-    "css-modules-typescript-loader": "^2.0.4",
+    "css-loader": "^7.1.2",
+    "terriajs-typings-for-css-modules-loader": "^2.5.2",
```

Comment:
Non-alphabetical sorting, so this package will be shuffled down to the other terriajs-packages the next time someone runs `yarn add`.